keytableprinter/keytableprinter.go

```diff
@@ -2,94 +2,179 @@ package keytableprinter
 
 import (
 	"fmt"
+	"sort"
 	"strings"
 
 	"github.com/fluidkeys/fluidkeys/colour"
 	"github.com/fluidkeys/fluidkeys/pgpkey"
 )
 
+type column = []string
+type row = []string
+
 const gutter = "  "
+const divider = "---"
 
-var headers = [3]string{
-	"Email address",
-	"Created",
-	"Next rotation",
+var header = row{
+	colour.Info("Email address"),
+	colour.Info("Created"),
+	colour.Info("Status"),
 }
 
+var placeholderDividerRow = row{divider, divider, divider}
+
 func Print(keys []pgpkey.PgpKey) {
```

Comment:
This is the only public interface, right? It might be clearer to move it to the top, above all the unexported stuff.